_dashboards/dql.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: default
+title: Dashboards Query Language
+nav_order: 99
+---
+
+# Dashboards Query Language
+
+Similar to the [Query DSL]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/index) that lets you use the HTTP request body to search for data, you can use the Dashbaords Query Language (DQL) in OpenSearch Dashboards to search for data and visualizations.
+
+For example, if you want to see all visualizations of visits to a host based in the US, enter `geo.dest:US` into the search field, and Dashboards refreshes to display all related data.
+
+Just like query DSL, DQL comes in a handful of varieties, so use whichever best fits your use case.
+
+---
+
+#### Table of contents
+1. TOC
+{:toc}
+
+---
+
+## Terms query
+
+The most basic query is to just specify the term you're searching for.
+
+```
+host.keyword:www.example.com
```

Comment:
I assume each period-separated string is a document field? Can we be clear about that?